### DISCUSSION TITLE
onClick not work after add new own plugin

### DISCUSSION THREAD

**Discussion by SantiagoMena** [freshframework/fresh#1051] [Q&A]
Hi, first, sorry for my english.

I `add new plugin` for inject Google Tag Manager into my deno fresh project, so i add:

// **plugins/gtm/gtm.ts**
import { Plugin } from "$fresh/server.ts";

export default function gtm(): Plugin {
  return {
    name: "gtm",
    entrypoints: { "tag": import.meta.resolve("./tag.js") },
    render(ctx) {
      const res = ctx.render();
      const scripts = [];
      
      if (res.requiresHydration) {
        scripts.push({ entrypoint: "tag", state: [] });
      }

      return {
        scripts
      };
    },
  };
}

// **plugins/gtm/tag.js**
// any custom js, ex:
`alert();`

// **main.ts**
...
...
...
import { start } from "$fresh/server.ts";
import manifest from "./fresh.gen.ts";
import "https://deno.land/std@0.145.0/dotenv/load.ts"; 
import twindPlugin from "$fresh/plugins/twind.ts";
import twindConfig from "./twind.config.ts";

import gtmPlugin from "./plugins/gtm/gtm.ts";

await start(manifest, { port: 3000, plugins: [twindPlugin(twindConfig), gtmPlugin()]});

`After` i add the plugin` all onclick tag functions not work`.

// **islands/Header.tsx**
......
<a onClick={() => setHelpActive(!helpActive)} ">Cerrar</a>

I hope it is understood, and can reproduce the error. Stay tuned to your comments

**Comment by marvinhagemeister** [comment]
Closing. If this is still an issue I'd recommend opening an issue.
